src/scripts.js

```diff
@@ -95,24 +116,32 @@ function renderBookings() {
   bookingsSection.innerHTML += `<p>Total Spent: $${hotel.getTotal().toFixed(2)}` 
 }
 
-function showAvailable(date) {
+function showAvailable() {
+  const date = dateInput.value.split('-').join('/');
+  console.log(date)
+  const type = typeSelect.value;
+  console.log(hotel.bookings)
+  hotel.findAvailable(date);
+  filterAvailable(type);
   if(date && hotel.availableRooms.length !== 0) {
     availableSection.innerHTML = ''
```

Comment:
You may want to consider adding a message here letting the user know there aren't any rooms available and to try a different search.